themes/digital.gov/static/js/toc.js

```diff
@@ -16,7 +16,6 @@ jQuery(function ($) {
 
 	function format_toc(hash){
 
-
 		// checks if it is a mobile browser
 		if (mobile_check() == true) {
 			var num = $('nav#TableOfContents ul:first-child > li').size();
```

Comment:
Is this file still needed? I thought we were replacing this.

Reply:
There are other page that still use this hugo variable. This can be removed when we replace it on the other content types. Will make a note to remove this file.

Reply:
Can you create a trello card to bring in-page nav to other pages too?